lms/static/sass/views/_learner-profile.scss

```diff
@@ -307,8 +308,244 @@
             border: 2px dashed $link-color;
             .u-field-title,
             i {
-              color:  $link-color;
+                color: $link-color;
+            }
+        }
+    }
+
+    .badge-paging-header {
+        padding-top: $baseline;
+    }
+
+    .pagination {
```

Comment:
@nasthagiri A pagination framework exists, but it is not usable here. The SASS only exists in the CMS, and my attempts to move it to a common space proved fruitless. This is a modified copy of what is there.